### DISCUSSION TITLE
Win 11 22H2

### DISCUSSION THREAD

**Discussion by AYL-G** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#86] [General]
Anyone heard about the new issues in windows 11 22H2?
I wanted to get the update to finally get  back the official drag and drop feature, however it is said that there are many issues and I don't know if I should install it.

**Comment by HerMajestyDrMona** [comment]
I was using the BETA builds for a few months now. Except of some "normal" issues ("normal" for Microsoft standards, shouldn't exist in a productive OS), like taskbar freeze, window preview not working from time to time, etc. I didn't experience any serious issues that would stop me from using it. But again, it also depends on your hardware, drivers and other programs running. So it's really hard to answer... I would recommend you to try it, but I don't want to get blames in case when something breaks :)

**Reply by AYL-G** [reply]
I found out that there's an issue for Nvidia and Intel users. Better wait for a fix

**Reply by HerMajestyDrMona** [reply]
Well, I use an Nvidia GPU on my Desktop PC and didn't notice any issues. Most of these "news" online are written for a BOOM, while the problems affect a very low percentage of users.

**Reply by ElitePheonix009** [reply]
> Well, I use an Nvidia GPU on my Desktop PC and didn't notice any issues. Most of these "news" online are written for a BOOM, while the problems affect a very low percentage of users.

Nvidia has fixed the issue.

**Reply by HerMajestyDrMona** [reply]
> > Well, I use an Nvidia GPU on my Desktop PC and didn't notice any issues. Most of these "news" online are written for a BOOM, while the problems affect a very low percentage of users.
> 
> Nvidia has fixed the issue.

As I mentioned in my earlier comment, I was using the BETA build of Windows 11 22H2 for a few months now. It all worked fine on the Nvidia GPU (and integrated AMD in my laptop). It's obvious that problems don't affect all users, just a small permille of users. It might still be a high number considering all Windows users, but it also means that most of users should not worry about such articles.
